### DISCUSSION TITLE
详解 Kubernetes Job 和 CronJob 的实现原理 · /kubernetes-job-cronjob

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#553] [Announcements]
https://draveness.me/kubernetes-job-cronjob

**Comment by talway** [comment]
你好，你的博客文章图片全部没有了，每个文章的图片都没有，看不到图示

**Comment by draveness** [comment]
> @talway
> 你好，你的博客文章图片全部没有了，每个文章的图片都没有，看不到图示

没有问题啊

**Comment by jlon** [comment]
写的很好，加油！

**Comment by qhh0205** [comment]
什么时候出 k8s 开发系列文档？

**Comment by sadwxqezc** [comment]
`{% include related/distributed-system.md %}` 这块崩了

有个疑问，一次性从ApiServer中取出所有的CronJob，量是不是太大了

**Comment by draveness** [comment]
> `{% include related/distributed-system.md %}` 这块崩了
> 
> 有个疑问，一次性从ApiServer中取出所有的CronJob，量是不是太大了

看有多少 CronJob 了

**Comment by sadwxqezc** [comment]
> > `{% include related/distributed-system.md %}` 这块崩了
> > 有个疑问，一次性从ApiServer中取出所有的CronJob，量是不是太大了
> 
> 看有多少 CronJob 了

对，如果数量过大，内存可能就炸了

**Comment by draveness** [comment]
> > > `{% include related/distributed-system.md %}` 这块崩了
> > > 有个疑问，一次性从ApiServer中取出所有的CronJob，量是不是太大了
> > 
> > 
> > 看有多少 CronJob 了
> 
> 对，如果数量过大，内存可能就炸了

是会有这个问题，所有的 Controller 应该都有这个问题，这也是 Kubernetes 可伸缩性的瓶颈之一
